sdk/README.md

```diff
@@ -0,0 +1,99 @@
+# Azure SDK Libraries
```

Comment:
This is incomplete and would be unmaintainable over time as the number of libraries grows.  It has high potential for getting stale and I don't believe there's much value offered for contributors.